setup.cfg

```diff
@@ -70,6 +70,8 @@ invenio_base.api_blueprints =
     invenio_stats = invenio_stats.views:blueprint
 invenio_search.templates =
     invenio_stats = invenio_stats.templates:register_templates
+invenio_search.index_templates =
```

Comment:
any way of installing one or the other, not both? maybe we need to deprecate `invenio_search.templates`?